zips/zip-0032.rst

```diff
@@ -397,18 +402,24 @@ $\mathsf{MKGh}^\mathsf{Context}(\mathsf{IKM}) \rightarrow (\mathsf{sk}_m, \maths
 Hardened-only child key derivation
 ----------------------------------
 
-$\mathsf{CKDh}^\mathsf{Context}((\mathsf{sk}_{par}, \mathsf{c}_{par}), i)$ $\rightarrow (\mathsf{sk}_i, \mathsf{c}_i)$ :
+As well as the integer child index $i$, the child key derivation function defined here supports a
+byte sequence $\mathsf{tag}$ and a $\mathsf{full\_width\_leaf}$ flag ($\!0$ representing $\mathsf{false}$,
```

Comment:
This is confusing; this appears to be stating that the type of `full_width_leaf` is a byte with a boolean interpretation, but a boolean _value_ is passed for it below. Either specify this as a boolean, or pass a byte value in the function invocation.

Reply:
In the Zcash protocol spec and in Zcash specs in general, 0 is false and 1 is true. So there's no need for an explicit conversion from true/false to 0/1.